src/js/options.js

```diff
@@ -362,6 +362,16 @@ function removeWhitelistDomain(event) {
   reloadWhitelist();
 }
 
```

Comment:
Let's document this workaround function please.

Reply:
Which workaround function do you mean? I didn't write anything related to `reloadWhitelist` - did you mean `getScrollbarWidth`?

Reply:
Yes, sorry it's not clear in this view.